src/OpenTelemetry.Abstractions/Utils/Collections.cs

```diff
@@ -60,12 +60,5 @@ public static string ToString<T>(IEnumerable<T> list)
 
             return sb.ToString();
         }
-
-        public static bool AreEquivalent<T>(IEnumerable<T> c1, IEnumerable<T> c2)
```

Comment:
I recall we wanted to get rid of the whole file, did we?

Reply:
issue is still active =)